### PR TITLE
Preparing release 0.17.0 of trace

### PR DESCRIPTION
**Pull request by liyanhui1228**
@dhermes I need to bump the version number from 0.16.0 to 0.17.0 to avoid breaking the OpenCensus package which is depending on `google-cloud-trace>=0.16.0, <0.17dev`. Could you do a release for the trace package? Thanks a lot!

### CODE REVIEW

File: trace/CHANGELOG.md [hunk 1]
```diff
@@ -12,3 +12,13 @@
...
 
 PyPI: https://pypi.org/project/google-cloud-trace/0.16.0/
+
+## 0.17.0
+
+### Notable Implementation Changes
+
+- Default to use Stackdriver Trace V2 API if calling `from google.cloud import trace`.
+  Using V1 API needs to be explicitly specified in the import.(#4437)
+- Comparing `datetime.datetime`-s in trace unit tests. (#4323)
```
Comment (dhermes) [inline trace/CHANGELOG.md]: @liyanhui1228 This doesn't belong in release notes (but thanks for being thorough)

File: trace/CHANGELOG.md [hunk 2]
```diff
@@ -12,3 +12,13 @@
   on `google-api-core` (#4221, #4280)
 
 PyPI: https://pypi.org/project/google-cloud-trace/0.16.0/
+
+## 0.17.0
```
Comment (dhermes) [inline trace/CHANGELOG.md]: This should go **above** the already existing release notes (I think?)

Comment (liyanhui1228) [reply trace/CHANGELOG.md]: Done.

File: trace/CHANGELOG.md (hunk 1 above)
Comment (liyanhui1228) [reply trace/CHANGELOG.md]: Removed.

**Comment by liyanhui1228** [comment]
Thanks for your review, I have updated the release.rst and addressed the comments. I just checked that I can create the tag after this is merged.

**Comment by liyanhui1228** [comment]
@dhermes I don't have the account to push packages to Pypi, will need your help. Thx!

**Comment by dhermes** [comment]
@liyanhui1228 The [build][1] will do that.

[1]: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/4618

**Comment by liyanhui1228** [comment]
@dhermes Great!

**Pull request merged**